packages/common/src/grpc-service.js

```diff
@@ -161,6 +161,11 @@ function GrpcService(config, options) {
 
   this.grpcMetadata = new grpc.Metadata();
 
+  this.grpcMetadata.add('User-Agent', [
+    config.packageJson.name.replace('@google-cloud', 'gcloud-node'),
```

Comment:
This would make the header appear as so `gcloud-node/bigtable/v0.3.0`, correct? Do we want the service name in there?